doc/contributing.md

```diff
@@ -9,4 +9,4 @@ This guide outlines ways to get started with contributing to Ruby:
   to change Ruby's documentation, code, test suite, or standard libraries
 * [Making changes to Ruby standard libraries](contributing/making_changes_to_stdlibs.md): How to build, test, and contribute to Ruby standard libraries
 * [Making changes to Ruby documentation](contributing/documentation_guide.md): How to make changes to Ruby documentation
-* [Benchmarking Ruby](https://github.com/ruby/ruby/tree/master/benchmark#make-benchmark): How to benchmark Ruby
+* [Benchmarking Ruby](https://github.com/ruby/ruby/tree/master/benchmark#user-content-make-benchmark): How to benchmark Ruby
```

Comment:
The old link works while this new link is broken for me. Is this an error?

Reply:
I'll look into it.

Reply:
This link with its fragment works for me in Edge, FireFox, and Bravo.  Does it not go to the page at all?

Reply:
Ah it looks like there is a bug in the GitHub redesign preview. Both of these links work for me in the old design. Does the old link not work for you?

Reply:
Old link works for me.

Reply:
If the old link works then we shouldn't change it. The old link is also the one that GitHub suggests to use.

Reply:
> If the old link works then we shouldn't change it. The old link is also the one that GitHub suggests to use.

This is one where some browsers do a soft match;  the link as it stands has the full fragment.  I thought we'd agreed to harden these.  Btw, what browser or application are you using where this is broken?

Reply:
> This is one where some browsers do a soft match; the link as it stands has the full fragment.

The original link is the one suggested by GitHub when you click the chain icon next to the heading.

![image](https://github.com/ruby/ruby/assets/15860699/fdd3190c-435d-460e-b24a-6dfae4213bd0)

We should use the one suggested by GitHub.

> Btw, what browser or application are you using where this is broken?

Turns out this wasn't a browser issue. I opted into GitHub's redesign beta, which has a bug where the links don't quite work.

Reply:
Restored.